dex/testing/polygon/harness.sh

```diff
@@ -323,11 +346,78 @@ cat > "${NODES_ROOT}/test_block10_hash.txt" <<EOF
 ${TEST_BLOCK10_HASH}
```

Comment:
This already happened in the previous PR, but for me when the 10th block hash is being retrieved, the 10th block has not yet been mined. Works fine on ETH though.

Reply:
Mmkay. The mine-alpha script for polygon doesn't seem to always work right, and mining on polygon is touchy anyway. It's difficult to produce a small number of blocks and sometimes no blocks are produced, but the mine-alpha script still exits. Will look for a solution.